spec/services/decisions/client_decision_tree_spec.rb

```diff
@@ -88,6 +122,8 @@
 
       context 'when the applicant has a passporting benefit' do
         context 'has correct next step' do
+          let(:benefit_type) { BenefitType::UNIVERSAL_CREDIT.to_s }
```

Comment:
We could move this let to the immediate outer context to avoid repetition in all sub-contexts?

Reply:
Good plan 👍